### PR TITLE
Added code explanation to welcome.html

### PR DESCRIPTION
**Pull request by HugoLazaro**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,4 +14,6 @@ Some ideas for obtaining a :gift: if you are the first that:
 user name | NIA | GitHub Action |Score
 ----------|-----|---------------|-----
 [UNIZAR-30246-WebEngineering](https://github.com/UNIZAR-30246-WebEngineering/lab1-git-race) |30246 | [![Build Status](https://github.com/UNIZAR-30246-WebEngineering/lab1-git-race/actions/workflows/ci.yml/badge.svg)](https://github.com/UNIZAR-30246-WebEngineering/lab1-git-race/actions/workflows/ci.yml)
+[HugoLazaro](https://github.com/HugoLazaro/lab1-git-race) |801758 | [![Build Status](https://github.com/HugoLazaro/lab1-git-race/actions/workflows/ci.yml/badge.svg)](https://github.com/HugoLazaro/lab1-git-race/actions/workflows/ci.yml)
+(https://travis-ci.org/HugoLazaro/lab1-git-race)
```
Comment (fjlopez) [inline README.md:18]: Remove this line

**Review by fjlopez — CHANGES_REQUESTED** [review]

**Comment by fjlopez** [comment]
Remove the line and leave an empty line. Next, I will accept your PR.

File: README.md (hunk 1 above)
Comment (HugoLazaro) [reply README.md:18]: I have removed the last line

**Comment by fjlopez** [comment]
My comment in the review was not very accurate. Remove line 18 (it is useless because your data is in line 17) and replace it with an empty line. Ending text documents with an empty line is a good practice.

File: README.md (hunk 1 above)
Comment (HugoLazaro) [reply README.md]: Is this correct now? I have replaced line 18 with an empty line.

**Pull request merged**